### DISCUSSION TITLE
[security] fail2ban

### DISCUSSION THREAD

**Discussion by tuxnetDE** [paperless-ngx/paperless-ngx#407] [Feature Requests] [answered]
Is it possible to integrate fail2ban ?

**Comment by rum0s** [comment]
Adding failed auth attempts with corresponding IPs into logs would be great for setting up crowdsec or fail2ban.

**Comment by stevenengland** [comment]
Would love that one too.

**Comment by OddSquirrel** [comment]
Couldn't agree more. The lack of it is pretty much the reason why I don't expose it to the outside at all.

**Reply by maxime1992** [reply]
If you want to, you could have a look into Authelia. This would let you add double auth in front of all/some of your self hosted services. And use fail2ban with Authelia. I use it with SWAG and it's made my life a lot simpler

**Comment by shamoon** [comment, marked as answer]
Forgot to update here, see #2359 

(but I do agree RP is the better solution)
